File: 2239-find-closest-number-to-zero/2239-find-closest-number-to-zero.py
class Solution(object):
    def findClosestNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ans = nums[0]
        for i in nums:
            if abs(i) < abs(ans) or (abs(i) == abs(ans) and i > ans):
                ans = i
        return ans

File: 2239-find-closest-number-to-zero/2239-find-closest-number-to-zero.java
class Solution {
    public int findClosestNumber(int[] nums) {
        int n = nums.length;
        int ans = nums[0];
        for(int i= 0; i<n; i++){
            if(Math.abs(nums[i])<Math.abs(ans) || (nums[i] > ans && Math.abs(nums[i])==Math.abs(ans)))
                ans = nums[i];
        }
        return ans;
    }
}